src/components/AlertMessage/AlertMessage.jsx

```diff
@@ -0,0 +1,15 @@
+import { Alert, AlertTitle } from "@mui/material";
+
+const AlertMessage = ({ alertMsg, severity }) => {
+  const message = severity === "error" ? "Something went wrong - " : "";
+  return (
+    <Alert severity={severity}>
+      <AlertTitle>Oops!</AlertTitle>
+      <p>
+        {message} <strong>{alertMsg}</strong>
+      </p>
```

Comment:
I think using material ui was a good idea here. Would be interesting to see if you can mimic this behaviour